GPU/Common/TextureDecoder.h

```diff
@@ -49,7 +49,7 @@ void DoUnswizzleTex16Basic(const u8 *texptr, u32 *ydestp, int bxc, int byc, u32
 #define DoReliableHash32 XXH32
 #define DoReliableHash64 XXH64
 
-#ifdef _M_X64
+#ifdef _M_X64 || defined(ARM64)
```

Comment:
This is not correct, you have to write it ```#if defined(_M_X64) || defined(ARM64)``` , you can't mix #ifdef with expressions...

Reply:
Yes, oops, I'll update.